# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: devtools/server/connectors/js-window-actor/moz.build
# Commit: 895cf08432c4
# Full Hash: 895cf08432c40fe371ff83e846cb161430c6be4b
# Author: Nicolas Chevobbe <nchevobbe@mozilla.com>
# Date: 2020-10-27 04:41:26
# Regressor Bug: 1633712
# File Overlap Count: 1
# Description:
#   Bug 1633712 - [devtools] Create WorkerTargets as soon as possible. r=ochameau,devtools-backward-compat-reviewers.
#   
#   This patch adds support for dedicated worker targets in the Watcher actor.
#   Shared and Service workers are not handled yet.
#   
# ==============================================================================

diff -r 2ce79a9ed2d5 -r 895cf08432c4 devtools/server/connectors/js-window-actor/moz.build
--- a/devtools/server/connectors/js-window-actor/moz.build	Mon Oct 26 12:52:26 2020 +0000
+++ b/devtools/server/connectors/js-window-actor/moz.build	Mon Oct 26 12:53:43 2020 +0000
@@ -7,5 +7,7 @@
 DevToolsModules(
     'DevToolsFrameChild.jsm',
     'DevToolsFrameParent.jsm',
+    'DevToolsWorkerChild.jsm',
+    'DevToolsWorkerParent.jsm',
     'WindowGlobalLogger.jsm',
 )